agimus_controller_ros/agimus_controller_ros/agimus_controller.py

```diff
@@ -377,7 +378,7 @@ def run_callback(self, *args) -> None:
                 )
                 return
         if self.params.publish_debug_data:
-            start_compute_time = self.get_clock().now()
+            start_compute_time = time.perf_counter()
```

Comment:
Please add a comment so that someone does not revert that to ROS time.

My guess is
```suggestion
            # Do not use ROS time here because we want to measure the real computation time
            # even when using simulation time
            start_compute_time = time.perf_counter()
```